synctl

```diff
@@ -1577,6 +1577,27 @@ class PatchSyntheticTest(SyntheticTest):
             return
         self.__patch_a_synthetic_test(self.test_id, json.dumps(payload))
 
+    def patch_ping_operation(self, method: str = "GET"):
+        """operation"""
+        valid_methods = ["GET", "HEAD", "POST", "PUT", "DELETE",
+                         "CONNECT", "OPTIONS", "TRACE", "PATCH"]
+        payload = {"configuration": {"operation": ""}}
+        if method is not None and method.upper() in valid_methods:
+            payload["configuration"]["operation"] = method.upper()
+            self.__patch_a_synthetic_test(self.test_id, json.dumps(payload))
+        else:
+            print(f"{method} is not allowed")
+            return
+
+    def patch_mark_synthetic_call(self, markSyntheticCall):
```

Comment:
markSyntheticCall is boolean type, verify it is a valid value from `false`, `False`, `True`, and `true`. and throw an error if it is not correct.